core/workspace_svg.js

```diff
@@ -1752,6 +1744,54 @@ Blockly.WorkspaceSvg.prototype.removeToolboxCategoryCallback = function(key) {
   this.toolboxCategoryCallbacks_[key] = null;
 };
 
+/**
+ * Look up the gesture that is tracking this touch stream on this workspace.
+ * May create a new gesture.
+ * @param {!Event} e Mouse event or touch event
+ * @return {Blockly.Gesture} The gesture that is tracking this touch stream,
+ *     or null if no valid gesture exists.
+ */
+Blockly.WorkspaceSvg.prototype.getGesture = function(e) {
+  var isStart = (e.type == 'mousedown' || e.type == 'touchstart');
+
+  var gesture = this.currentGesture_;
+  if (gesture) {
+    if (isStart && gesture.hasStarted()) {
+      console.warn('tried to start the same gesture twice');
```

Comment:
This would happen from a series of events, possibly involving leaving the tab and losing a mouse up.  I don't think I can provide sufficient information to debug it from the event handler.  Instead I'm trying to recover from the failure while warning the developer (quite likely me) that some sequence of events caused a problem, and they should try to reproduce it.